VideoLocker/res/menu/change_mode.xml

```diff
@@ -1,10 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
 <menu xmlns:android="http://schemas.android.com/apk/res/android">
      <item android:id="@+id/change_mode_video_only"
-              android:icon="@drawable/change_mode"
-              android:title="Video Only"
-              android:orderInCategory="100" />
+           android:titleCondensed="mode_video_only"
```

Comment:
@aleffert it does not need localization.  basically just a hack to get menuitem id